### ISSUE TITLE
Normalization implementation

### ISSUE THREAD

**Issue by sunwell1994** [JusperLee/Conv-TasNet#30]
The normalization implementation is interesting here compared to the paper version and Yi Luo's implementation. 

Groupnorm: while Yi Luo used the nn.GroupNorm directly, this repo implement a new version? What is the difference between them?

cLN: seems this one is not cumulative.  Both cumulative version and simple layerwise norm perform terribly.

**Comment by JusperLee** [comment]
For the difference between this repository and Yi Luo in global layer normalization: the settlement formula of torch.nn.GroupNorm is consistent with this repository. The reason I rewrote it myself is that I didn't know this function existed.

For the cLN issue, you are right, my implementation of cLN is wrong, you can submit the updated code if you are interested. But currently, most researchers using the repository are mainly concerned with the noncasual model. The specific implementation can be found in the following code.

https://github.com/naplab/Conv-TasNet/blob/e66d82a8f956a69749ec8a4ae382217faa097c5c/utility/models.py#L8-L45

**Comment by sunwell1994** [comment]
Thanks for your reply. It seems that various implementations of cLN have similar performce.

**Comment by JusperLee** [comment]
Yes, it can be considered that way, but different implementations may still have deviations.
